templates/submission/form/step1.tpl

```diff
@@ -22,18 +22,32 @@
 {include file="controllers/notification/inPlaceNotification.tpl" notificationId="submitStep1FormNotification"}
 
 {fbvFormArea id="submissionStep1"}
-	<!-- Author user group selection (only appears if user has > 1 author user groups) -->
-	{if count($authorUserGroupOptions) > 1}
-		{fbvFormSection label="submission.submit.userGroup" description="submission.submit.userGroupDescription"}
-			{fbvElement type="select" id="authorUserGroupId" from=$authorUserGroupOptions translate=false size=$fbvStyles.size.SMALL}
+	<!-- If no existing roles, show available author roles to choose from -->
+	{if $noExistingRoles}
+		{fbvFormSection label="submission.submit.availableUserGroups" description="submission.submit.availableUserGroupsDescription" list=true required=true}
```

Comment:
Can we add a check here if `count($userGroupOptions) < 1`? If it's allowed under GDPR, we can add a hidden field. If we require a consent confirmation, let's make it a required checkbox:

> Yes, I would like to be contacted as the corresponding contact for this submission.

If the checkbox is not completed:

> "You must agree to be contacted as the corresponding contact for this submission."

(Or, if it's allowed by GDPR, the best would be to have no opt-in, but just a message at the bottom that says: "By making a submission, you agree to be contacted as the corresponding contact for this submission.")